__unported__/l10n_it_DDT_webkit/__openerp__.py

```diff
@@ -31,6 +31,7 @@
     """,
     'author': 'Agile Business Group',
     'website': 'http://www.agilebg.com',
+    'license': 'GPL-3 or any later version',
```

Comment:
Header says GPL due to my inattention (wrong copy and paste).
I made https://github.com/yvaucher/l10n-italy/pull/1